.github/ISSUE_TEMPLATE/feedback.yml

```diff
@@ -1,12 +1,12 @@
-name: FinOps Use Case Feedback
```

Comment:
If we're going to collect non-FinOps use cases (per line 21 below), then we should rename this and scrub other lines to not be FinOps-specific.

Reply:
@shawnalpay I see that you have proposed specific changes below. I will close this one and work on the proposal.